Assets/HoloToolkit-Examples/Input/Prefabs/ControllerRect.prefab

```diff
@@ -0,0 +1,82 @@
+%YAML 1.1
```

Comment:
There are a couple files added back that were moved in the base Toolkit. This one and MicrophoneHelperSample should be removed. It might be a good idea to rebase before we merge this, since it looks like there might have been a merge somewhere that added these removed files.